index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Studio Ghibli </title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+        <script src="./index.js" async ></script>
+        <link href="./index.css" rel="stylesheet">
+    </head>
+        <header>
+            <img src="https://upload.wikimedia.org/wikipedia/en/thumb/c/ca/Studio_Ghibli_logo.svg/1200px-Studio_Ghibli_logo.svg.png" alt="Studio_Ghibli_logo">
+            <h1>Ghibli Review App</h1>
+        </header>
+        <section>
+            <form action="submit" id="filmQuery">
+                <select id="filmList">
```

Comment:
You should put your select outside of the form tag. Your select should work right after the user has picked the movie, not make the user do an additional action of clicking the submit button.